pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
+	instanceClass := getInstanceClass(region)
+	return fmt.Sprintf("%s_D4s_v3", instanceClass)
+}
+
+func ControlPlaneInstanceType(region string) string {
```

Comment:
this is missing comments for public function https://blog.golang.org/godoc-documenting-go-code

Reply:
Fixed.